controllers/community.js

```diff
@@ -13,7 +13,7 @@ router.get('/', (req, res, next) => {
 		}
 		return Promise.resolve(template);
 	}).then(template => res.render(template, {
-		title: 'Mitmachen',
+		title: 'Jetzt mitmachen!',
```

Comment:
Can you please add this to the localization en.json and de.json?

Reply:
> @dominikbruechner n21-schoolcloud-logo.svg and th-schoolcloud-logo.svg is theming content, is the location well or must it changed?

I think its fine, as it is just displayed on the "Registrieren" Overview Page